packages/@lwc/engine-dom/src/renderer.ts

```diff
@@ -17,6 +17,16 @@ import {
 import { Renderer } from '@lwc/engine-core';
 
 const globalStylesheets: { [content: string]: true } = create(null);
+
+if (process.env.NODE_ENV === 'development') {
+    // @ts-ignore
+    window.__lwcResetGlobalStylesheets = () => {
+        for (const key of Object.keys(globalStylesheets)) {
```

Comment:
You could create a new object `create(null)` and assign it.

Reply:
Avoid having to turn a `const` into a `let`?

Reply:
That is a valid reason 👍 . I am generally not a fan of changing app code to suit testing.